docs/kubernetes/config_examples/f5-k8s-ingress-fanout.yaml

```diff
@@ -4,22 +4,23 @@ metadata:
   name: ing-fanout
   namespace: default
   annotations:
-    # IP address of a BIG-IP pool member
+    # Provide an IP address from the external VLAN on your BIG-IP device;
+    # a hostname resolvable by DNS; or use the default IP
```

Comment:
If users want to use the DNS lookup feature, the hostname that gets resolved is the first "host" under "spec:rules" below, not the annotation. (This applies to all comments that you have added in these Ingress examples.)